### DISCUSSION TITLE
I haven't received the GitHub Student Developer Pack after 5 days

### DISCUSSION THREAD

**Discussion by NHBakli** [community/community#140252] [GitHub Education]
### Hi everyone,

Hy, I haven't received the education pack after 5 days. How to resolve this probleme ?

**Comment by InboraStudio** [comment]
We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
We will be locking this discussion. If you haven't received your benefits after approval within 6 days please raise a new discussion so we can address it 👍

**Reply by NHBakli** [reply]
I consulted the FAQ but I was accepted on September 24, that's more than 6 days

**Comment by davevad93** [comment]
Hi @NHBakli ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @NHBakli was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
